CHANGES.rst

```diff
@@ -1,6 +1,20 @@
 
 Changelog
 =========
+--------------------
+3.21 (2024-09-17)
```

Comment:
should this be 3.21.0  ? like [3.20.0](https://github.com/aws/aws-elastic-beanstalk-cli/releases/tag/3.20.0)

Reply:
👍